lib/polarshift/test_suite.rb

```diff
@@ -108,7 +108,8 @@ def commit!(test_case)
         if self.current_test_record.start_scenario_for! test_case
           return true
         else
-          raise "logic error: starting test case that is not part of current test record"
+          puts "logic error: starting test case '#{test_case}' that is not part of current test record\n"
```

Comment:
@pruan-rht should the `puts` also be replaced with `logger`?

Reply:
@jhou1 there is no logger defined in the file

Reply:
we can add the logger class at a later time

Reply:
OK, thanks